setup/options/options_foss.rb

```diff
@@ -0,0 +1,24 @@
+{
+  :type                        => 'foss',
+  :pre_suite                   => [
+    'setup/install_gatling/00_pre_install/05_initialize_helpers.rb',
```

Comment:
It looks like the `pre_install` and `post_install` steps are the same. Would it be possible / worthwhile to specify these in `options_common.rb` and just specify the unique steps in the `options_foss.rb` and `options_pe.rb`?

Reply:
Makes sense, and there isn't really that much duplication since it's just foss and pe.